bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -603,6 +609,59 @@ impl Room {
         let user_id = UserId::parse(&user_id)?;
         Ok(self.inner.get_suggested_user_role(&user_id).await?)
     }
+
+    pub fn default_power_levels(&self) -> RoomPowerLevels {
+        let levels = SdkRoom::default_power_levels();
+        RoomPowerLevels::from(levels)
+    }
+}
+
+#[derive(uniffi::Record)]
+pub struct RoomPowerLevels {
```

Comment:
Honestly, if it's ok to add this struct, then I would much rather use this and a `power_levels` fn over the existing implementation of `RoomPowerLevelChanges` coming out from `build_power_level_changes_from_current`.

Reply:
Is that struct needed if all you want is to reset things? Wouldn't a `Room::reset_power_levels()` method server you better?

Reply:
After looking at all the changes needed to make this work I think a `reset_power_levels` function may actually be way simpler. @pixlwave I remember we wanted to use this strategy for some reason, but I can't recall why. Do you remember it?

Reply:
Yeah: right now `build_power_level_changes_from_current` gives us a `RoomPowerLevelChanges` which has optional values so we wanted to be able to get a default value when one was `nil`. But… If we were to replace that with a simple `power_levels` fn, then a `reset_power_levels` would totally do instead.

Reply:
Ah, I see. We need the current power levels to be able to display them in the UI. Then I can try changing the implementation to retrieve the current one instead of the default one and add a reset function.

Reply:
I just added some changes to:

1. Return the current power level values of a Room, not only the overridden ones. This should unblock displaying them in the UI.
2. Reset those values.

Reply:
Nice, this looks tidy! I think we can probably remove this [From<RoomPowerLevels>](https://github.com/matrix-org/matrix-rust-sdk/blob/a204b2994d4c5d58c4769e767f61fc7db7546e38/crates/matrix-sdk/src/room/power_levels.rs#L83-L110) now too.